### ISSUE TITLE
Chokk + Defensive Fire (v) game cancelled due to error

### ISSUE THREAD

**Issue by jokerking16** [PlayersCommittee/gemp-swccg-public#225]
https://gemp.starwarsccg.org/gemp-swccg/game.html?replayId=jokerking$0mergywogmhwge6q

It is not exactly clear which part of this combination caused the game cancelling error.

Chokk, armed with a Scout Blaster, was battled and played Defensive Fire (v).  We Need Your Help subtracted from the draw, resulting in a miss.  The subsequent chat message: Game was cancelled due to an error, the error was logged and will be fixed soon. appeared.

Error occurred on LS turn 4.

**Comment by MrFromMars** [comment]
https://gemp.starwarsccg.org/gemp-swccg/game.html?replayId=MrFromMars$ii1qj620695siw29

Happened again with defensive fire (v) and a different character set, different location. 

Error on LS turn 4.

**Comment by OrehRuoy** [comment]
https://gemp.starwarsccg.org/gemp-swccg/game.html?replayId=orehruoy$9yabta9sdcg6codl

Another time.
Last turn of game obviously.
Ls started fight. I used defensive fire to shoot Leia with Scout blaster. I added 1 with Maul to hit. I had an interrupt in hand that I could play after shooting and another start of battle interrupt but I don't think they have anything to do with it due to the other people having bugs.
Defensive fire (v) and scout blaster.
